lib/api/response/login_response.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'login_response.g.dart';
+
+@JsonSerializable()
+class LoginResponse {
+  @JsonKey(name: 'access_token')
+  String accessToken;
+  @JsonKey(name: 'token_type')
+  String tokenType;
+  @JsonKey(name: 'expires_in')
+  double expiresIn;
+  @JsonKey(name: 'refresh_token')
+  String refreshToken;
+  @JsonKey(name: 'created_at')
+  double createdAt;
```

Comment:
same, use config for snake case field name

```suggestion
  String accessToken;
  String tokenType;
  double expiresIn;
  String refreshToken;
  double createdAt;
```